Texture.py

```diff
@@ -1,163 +1,326 @@
-#FIXME: Move to xboxpy.NV2A.PGRAPH.Texture
+"""Provides functions for manipulating xbox textures."""
 
-from xboxpy import nv2a
+# pylint: disable=consider-using-f-string
```

Comment:
I prefer not to disable these linter warnings globally, and just let the linter complain so this file eventually does get cleaned up